mobile/lib/firebase_options.dart

```diff
@@ -63,6 +87,16 @@ class DefaultFirebaseOptions {
     iosBundleId: 'finance.get10101.app',
   );
 
+  static const FirebaseOptions ios_test = FirebaseOptions(
+    apiKey: 'AIzaSyC1DcfWnlt0BQMpX7_z3f0QPTgL1Po7zB8',
+    appId: '1:958970316277:ios:b85e19a35086fb681e0cbc',
+    messagingSenderId: '958970316277',
+    projectId: 'testnet-467c6',
+    storageBucket: 'testnet-467c6.appspot.com',
+    iosClientId: 'unset?!?',
```

Comment:
This field was not provided in the new firebase config for our test setup. I've set it to `unset?!?` on purpose so that we know what to look for if some firebase config ever fails. 
Maybe Mariusz knows why the new config did not include this field.